client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -99,18 +99,39 @@ const DataFilesToolbar = ({ scheme, api }) => {
     });
   };
 
+  const protectedFiles = [
+    '.bash_profile',
+    '.bashrc',
+    '.bash_history',
+    '.ssh',
+    'authorized_keys',
+    '.APPDATA',
+    '.Trash'
+  ];
+  const isProtected = selectedFiles.some(file =>
+    protectedFiles.some(protectedFile => protectedFile === file.name)
```

Comment:
Could be simplified a bit
```suggestion
    protectedFiles.includes(file.name)
```